### PR TITLE
add webpack configuration

### LINKED ISSUE #806: Cofigure webpack build development

### PR DESCRIPTION
**Pull request by mohitsuman**
Fix #806

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
```
Comment (jeffmaury) [inline webpack.config.js]: Microsoft ?

Comment (mohitsuman) [reply webpack.config.js]: Yeah the webpack configuration as shown in the official documentation for vscode extension has the Microsoft license and even vscode-java uses the same. So I preferred the same.

**Comment by mohitsuman** [comment]
> What's the exact purpose of this change ? What's the origin of this ? It's not clear at all for novices

Currently when we build the vscode extension for the release, it prompts us to use the webpack build as the size of the extension and no of files have increased. So we will be using webpack to compile these multiple small files into one big file and bundle them.

webpack is the preferred bundler for javascript & typescript application.
https://code.visualstudio.com/api/working-with-extensions/bundling-extension

**Comment by jeffmaury** [comment]
Unit test are failing, we need to fix that

**Comment by mohitsuman** [comment]
> It is not a priority for extension with not so many files.

When we do a build for release, it prompts to do so. It is a good use case for the future coming releases.

**Comment by mohitsuman** [comment]
Will fix this in next iteration, hence closing the PR.

**Pull request closed without merging**